plugins/Sid/CMakeLists.txt

```diff
@@ -1,51 +1,11 @@
 INCLUDE(BuildPlugin)
 
-INCLUDE_DIRECTORIES(resid)
+include_directories(resid/resid)
```

Comment:
This include directory logically belongs with the resid object files, and it would be nice to be able to simply link `sid` to `resid`. This can be done by creating an interface library, then putting the object files in its interface sources and the include directory in its interface include directories. Not tested, but it would look something like this:
```cmake
# In resid/CMakeLists.txt

add_library(resid_objects OBJECT
	# ...
)
add_library(resid INTERFACE)
target_sources(resid INTERFACE $<TARGET_OBJECTS:resid_objects>)
target_include_directories(resid INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/resid")

# In here

target_link_libraries(sid resid)
```
I used a similar trick when adding MSVC support to Zyn - see `plugins/ZynAddSubFx/CMakeLists.txt`.